app/views/simulator/edit.html.erb

```diff
@@ -489,7 +489,7 @@
   id="insertSubcircuitDialog"
   class="subcircuitdialog"
   title="Insert SubCircuit"
-></div>
+><div id="subCircuitNonEmptyDiaglog" style="height: fit-content;"></div><div id="subCircuitEmptyDialog"><span><p>Looks like there are no other circuits which doesn't have this circuit as a dependency. Create a new one!</p></span><a style="color:white;" class="btn btn-xs custom-btn--primary dropdown-item text-center logixButton pl-1" id="newCircuit">New Circuit +</a></div></div>
```

Comment:
Creating a new dialog for nonEmpty and Empty is over kill. Please use old dialog itself.